### PR TITLE
add Ui files and FXML files

### PR DESCRIPTION
**Pull request by airshhlay**
added FlashcardCard, FlashcardListPanel classes + their respective FXML files in resources/view
Subject to change as we still need to see how we want to present the flashcards. (eg. Question + Answer, just the Question, etc.)

### CODE REVIEW

File: src/main/java/cardibuddy/ui/FlashcardCard.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+import cardibuddy.model.flashcard.Flashcard;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+
+/**
+ * An UI component that displays information of a {@code deck}.
```
Comment (haveaqiupill) [inline src/main/java/cardibuddy/ui/FlashcardCard.java]: should be flashcard

File: src/main/java/cardibuddy/ui/FlashcardListPanel.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    private ListView<Flashcard> flashcardListView;
+
+    public FlashcardListPanel(ObservableList<Flashcard> flashcardList) {
+        super(FXML);
+        flashcardListView.setItems(flashcardList);
+        flashcardListView.setCellFactory(listView -> new FlashcardListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code deck} using a {@code flashcardCard}
```
Comment (haveaqiupill) [inline src/main/java/cardibuddy/ui/FlashcardListPanel.java]: should this be flashcard too

**Review by haveaqiupill — COMMENTED** [review]
Seems okay and can be merged. Have you tested this out?

File: src/main/java/cardibuddy/ui/FlashcardListPanel.java (hunk 2 above)
Comment (airshhlay) [reply src/main/java/cardibuddy/ui/FlashcardListPanel.java]: sorry!!! will change

File: src/main/java/cardibuddy/ui/FlashcardCard.java (hunk 1 above)
Comment (airshhlay) [reply src/main/java/cardibuddy/ui/FlashcardCard.java]: > Seems okay and can be merged. Have you tested this out?

nope have not tested out! cos i need a command to open the deck but we haven't done it yet. will probably try to replace all the deckCard with flashcardcard just to see if it works

**Pull request merged**